src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -189,9 +192,9 @@ impl Renderer for HtmlHandlebars {
         book.write_file("_FontAwesome/fonts/FontAwesome.ttf", theme::FONT_AWESOME_TTF)?;
 
         // Ace is a very large dependency, so only load it when requested
-        if book.uses_ace() {
+        if playpen_config.is_editable() {
             // Load the editor
-            let editor = editor::Editor::new(book.get_ace_path());
+            let editor = editor::Editor::new(playpen_config.get_editor());
             book.write_file("editor.js", &editor.js)?;
             book.write_file("ace.js", &editor.ace_js)?;
```

Comment:
`editor` is on the same level as `theme` and `renderer` right now, so I went with a similar naming convention. I could call it `playpen_editor`, but this depends on what other editors mdBook plans to have, if any. Like, MathJax, is that another type of editor? Is this perhaps not the right time to worry about this?

I could add some docs to the `Editor` definition, though, as right now it's not too obvious what kind of editor this is, and that way someone could just read that if they need to know about the editor.

I'm not sure about code paths. It's basically "if the playpen config says its editable, add all these other files/data in addition to the preceding files/data". I think making things clearer would require some major refactoring of `hbs_renderer` in general and I'm not sure what comments would add here (I'm not a huge fan of comments in general).

The situation with `hljs` is a bit odd since it's still sort of used by editable playpens for some CSS, and it will be used regardless of whether or not editability is enabled, it just won't be used for editable playpens specifically if editability _is_ enabled. So there really aren't two code paths that lead to different things, rather there's one code path that may additionally include the editor and then relies on JS magic to create the divergent behavior. I think the current code structure expresses that OK.